node/src/BaseClient.ts

```diff
@@ -77,6 +77,7 @@ import {
     createSMove,
     createSPop,
     createSRem,
+    createSUnionStore,
```

Comment:
This looks unrelated to the PR

Reply:
It was sorted alphabetically by the linter